airflow/migrations/versions/952da73b5eff_add_dag_code_table.py

```diff
@@ -38,6 +37,18 @@
 
 
 def upgrade():
+    from sqlalchemy.ext.declarative import declarative_base
+
+    Base = declarative_base()
+
+    class SerializedDagModel(Base):
+        __tablename__ = 'serialized_dag'
+
+        # There are other columns here, but these are the only ones we need for the SELECT/UPDATE we are doing
+        dag_id = sa.Column(sa.String(250), primary_key=True)
+        fileloc = sa.Column(sa.String(2000), nullable=False)
+        fileloc_hash = sa.Column(sa.BigInteger, nullable=False)
```

Comment:
Should the column type be the old one here?

Reply:
i.e. `Integer` ?

Reply:
Ignore me. I see we use the model after changing COlumn type

Reply:
Yup.